Documentation/gettingstarted/cni-chaining-aws-cni.rst

```diff
@@ -0,0 +1,131 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+*******
+AWS-CNI
+*******
+
+This guide explains how to set up Cilium in combination with aws-cni. In this
+hybrid mode, the aws-cni plugin is responsible for setting up the virtual
+network devices as well as address allocation (IPAM) via ENI. After the initial
+networking is setup, the Cilium CNI plugin is called attach BPF programs to the
+network devices set up by aws-cni to enforce network policies, perform
+load-balancing, and encryption.
+
+.. image:: aws-cni-architecture.png
+
+
+Setup Cluster on AWS
+====================
+
+Follow the instructions in the :ref:`k8s_install_eks` guide to set up an EKS
+cluster or use any other method of your preference to set up a Kubernetes
+cluster.
+
+Ensure that the `aws-vpc-cni-k8s <https://github.com/aws/amazon-vpc-cni-k8s>`_
```

Comment:
`aws-vpc-cni-k8s <https://github.com/aws/amazon-vpc-cni-k8s>`__

(Add 2 `_` at the end to make `aws-vpc-cni-k8s` a private reference otherwise we will have warnings if we create another href that uses the same name.)